test/TestOrchestrations.cs

```diff
@@ -115,14 +115,22 @@ public static string SayHelloInline([OrchestrationTrigger] DurableOrchestrationC
             await ctx.CallFunctionAsync(nameof(TestActivities.Throw), message);
         }
 
+        public static async Task OrchestratorGreeting([OrchestrationTrigger] DurableOrchestrationContext ctx)
+        {
+            string message = ctx.GetInput<string>();
+
+            // This throw happens in the implementation of an orchestrator.
+            await ctx.CallOrchestratorAsync(nameof(TestOrchestrations.SayHelloWithActivity), message);
```

Comment:
The comment here is confusing to me. What throw are you expecting?

Reply:
My bad, copy-past mistake.

---
In reply to: [138216528](https://github.com/Azure/azure-functions-durable-extension/pull/57#discussion_r138216528) [](ancestors = 138216528)